From 967cfc046d7403de9c423a06ec1b0caecdb74463 Mon Sep 17 00:00:00 2001
From: Paul Louvel <paul.louvel@bootlin.com>
Date: Thu, 30 Jul 2026 17:48:41 +0200
Subject: [PATCH 12530/15333] crypto: ecdsa - Fix typo in function
 documentation

Fix the misspelling of 'validate' in crypto_ecdh_shared_secret() and
ecc_is_pubkey_valid_partial() documentation.

Signed-off-by: Paul Louvel <paul.louvel@bootlin.com>
Signed-off-by: Herbert Xu <herbert@gondor.apana.org.au>
---
 include/crypto/internal/ecc.h | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/include/crypto/internal/ecc.h b/include/crypto/internal/ecc.h
index a4b48d76f53a..d67fe13a543a 100644
--- a/include/crypto/internal/ecc.h
+++ b/include/crypto/internal/ecc.h
@@ -149,7 +149,7 @@ int crypto_ecdh_shared_secret(unsigned int curve_id, unsigned int ndigits,
  * @curve:		elliptic curve domain parameters
  * @pk:			public key as a point
  *
- * Valdiate public key according to SP800-56A section 5.6.2.3.4 ECC Partial
+ * Validate public key according to SP800-56A section 5.6.2.3.4 ECC Partial
  * Public-Key Validation Routine.
  *
  * Note: There is no check that the public key is in the correct elliptic curve
@@ -166,7 +166,7 @@ int ecc_is_pubkey_valid_partial(const struct ecc_curve *curve,
  * @curve:		elliptic curve domain parameters
  * @pk:			public key as a point
  *
- * Valdiate public key according to SP800-56A section 5.6.2.3.3 ECC Full
+ * Validate public key according to SP800-56A section 5.6.2.3.3 ECC Full
  * Public-Key Validation Routine.
  *
  * Return: 0 if validation is successful, -EINVAL if validation is failed.
-- 
2.55.0

